elide-core/src/main/java/com/yahoo/elide/core/DataStoreTransaction.java

```diff
@@ -139,4 +139,54 @@ default void flush() {
                                                           Optional<Sorting> sorting, Optional<Pagination> pagination) {
         return collection;
     }
+
+    default <T> Object getRelation(
+            Object entity,
+            RelationshipType relationshipType,
+            String relationName,
+            Class<T> relationClass,
+            EntityDictionary dictionary,
+            Set<Predicate> filters
+    ) {
+        Object val = PersistentResource.getValue(entity, relationName, dictionary);
```

Comment:
To try and resurrect the previous discussion: why are we passing `entity` in as an `Object`. Why not just pass it in as a `PersistentResource` and call `getValue` on it directly?